content/FIDO2/FIDO2_WebAuthn_Developer_Guide/WebAuthn_Server_Registration.adoc

```diff
@@ -43,10 +43,11 @@ PublicKeyCredential<AuthenticatorAttestationResponse, ClientRegistrationExtensio
     jsonMapper.readValue(responseJson, new TypeReference<PublicKeyCredential<AuthenticatorAttestationResponse, ClientRegistrationExtensionOutputs>>(){});
 ----
 
-Pass the PublicKeyCredential as the argument to the RelyingParty finishRegistration method to validate the response. Be sure to remove the PubicKeyCredentialCreationOptions from the pending requests storage to prevent retries. The library performs a series of checks to ensure the registration ceremony was not tampered with. Including:
-Verifying the challenge is the same as the challenge that was sent
-Ensuring the origin was the origin expected
-Validating that the signature over the clientDataHash and the attestation using the certificate chain for that specific model of the authenticator.
+Pass the PublicKeyCredential as the argument to the RelyingParty finishRegistration method to validate the response. Be sure to remove the PubicKeyCredentialCreationOptions from the pending requests storage to prevent retries. The library performs a series of checks to ensure the registration ceremony was not tampered with, including:
+
+* Verifying the signature over the `clientDataHash` and the attestation using the certificate chain in the attestation statement
+* Optionally: verifying that the certificate chain is signed by a trusted certificate authority, as configured via the `metadataService` parameter of the `RelyingParty` class.
```

Comment:
Middle bullet point lost?

- Verifying that the origin was the origin expected

Reply:
Thanks! Fixed in ae46aac